### ISSUE TITLE
Plugin renaming my app

### ISSUE THREAD

**Issue by pekpon** [Telerik-Verified-Plugins/BarcodeScanner#28]
Im using this plugin with ionic.
On build, my app name is changed to "Barcode Scanner" always. How can I do to keep the original name?

Thanks!!

**Comment by EddyVerbruggen** [comment]
Oh dear that's not good. Android, right?

**Comment by pekpon** [comment]
thanks for reply!

Yes, its only on android...how can I share the app? What do you need? Full folder project? 

Thanks a lot!

**Comment by EddyVerbruggen** [comment]
I'll look into that further a bit.

Meanwhile, if you're able to build locally please check your `platforms/android/res/values/strings.xml` file. My Cordova CLI project has this (the app_name overrides the one passed in from the barcodescanner plugin):

```xml
<?xml version='1.0' encoding='utf-8'?>
<resources>
    <string name="app_name">HelloCordova</string>
    <string name="launcher_name">@string/app_name</string>
    <string name="activity_name">@string/launcher_name</string>
</resources>
```

**Comment by EddyVerbruggen** [comment]
Also interesting to know: is your testdevice running a non-English language?

**Comment by EddyVerbruggen** [comment]
Fixed in 2.1.2; users with non-English device language won't have an incorrect appname anymore.

**Comment by pekpon** [comment]
Wow! Really fast fix!!! Thanks a lot!
